docs/manage/troubleshoot/troubleshoot.md

```diff
@@ -89,6 +92,9 @@ If not follow the steps for the standalone version.
 {{% /tab %}}
 {{% tab name="macOS" %}}
 
+`viam-server` writes logs to STDOUT and does not store them in a file on your machine.
```

Comment:
stdout is a data stream. if a program is started in a terminal it's generally where stdout goes. If the program is started in the background or in a way that is survives the terminal window being killed stdout the output is just silently lost. So it doesn't go anywhere. At least I'm reasonably certain that is how it works. 
```suggestion
`viam-server` does not store logs in a file on your machine.
```
I can do this, any other change I can come up with makes this worse.